est/001.est

```diff
@@ -0,0 +1,22 @@
+date: 28-02-2016
+author: Dali Freire
+method: champions.pert
+scope:
+  1: Infrastructure setup
+  2: Design the operation commands
```

Comment:
@original-brownbear I'm talking about the implementation of what Rultor must do when find any of the [commands](http://doc.rultor.com/commands.html). Maybe the itens 2 and 3 must be merged... what do you think?